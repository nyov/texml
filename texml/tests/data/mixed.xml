<TeXML>
In |text| mode.
<TeXML>Still in |text| mode</TeXML>
<TeXML mode="math">
In |math| mode
<TeXML mode="text">In |text| mode inside math mode</TeXML>	
<TeXML>Still in |math| mode</TeXML>
</TeXML>
In |text| mode.
<math>a&lt;b</math> <dmath>c&gt;d</dmath>
</TeXML>
