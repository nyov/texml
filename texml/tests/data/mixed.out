In \textvert text\textvert  mode. Still in \textvert text\textvert  mode In |math| mode: |\hat .
In \textvert text\textvert  mode inside math mode Still in |math| mode
In \textvert text\textvert  mode: \textvert \^. $a<b$ $$c>d$$ $$ latex math: \hat{a}
latex text: \longleftarrow 
latex none: &#x777; $$
